stable_baselines3/ppo/ppo.py

```diff
@@ -282,19 +289,34 @@ def train(self, n_epochs: int, batch_size: int = 64) -> None:
         explained_var = explained_variance(self.rollout_buffer.returns.flatten(),
                                            self.rollout_buffer.values.flatten())
 
-        logger.logkv("n_updates", self._n_updates)
-        logger.logkv("clip_fraction", np.mean(clip_fraction))
-        logger.logkv("clip_range", clip_range)
-        if self.clip_range_vf is not None:
-            logger.logkv("clip_range_vf", clip_range_vf)
-
-        logger.logkv("approx_kl", np.mean(approx_kl_divs))
-        logger.logkv("explained_variance", explained_var)
-        logger.logkv("entropy_loss", np.mean(entropy_losses))
-        logger.logkv("policy_gradient_loss", np.mean(pg_losses))
-        logger.logkv("value_loss", np.mean(value_losses))
+        # Logs
+        logger.logkv("loss/entropy_loss", np.mean(entropy_losses))
+        logger.logkv("loss/policy_gradient_loss", np.mean(pg_losses))
+        logger.logkv("loss/value_loss", np.mean(value_losses))
+        logger.logkv("loss/approx_kl", np.mean(approx_kl_divs))
+        logger.logkv("loss/clip_fraction", np.mean(clip_fraction))
+        logger.logkv("loss/loss", loss.item())
+        logger.logkv("loss/explained_variance", explained_var)
         if hasattr(self.policy, 'log_std'):
-            logger.logkv("std", th.exp(self.policy.log_std).mean().item())
+            logger.logkv("loss/std", th.exp(self.policy.log_std).mean().item())
+        
+        logger.logkv("session/n_updates", self._n_updates, exclude='tensorboard')
+        logger.logkv("session/episode_reward", np.sum(self.rollout_buffer.rewards))
+        logger.logkv("input_info/discounted_rewards", np.sum(self.rollout_buffer.returns))
+        logger.logkv("input_info/advantage", np.mean(self.rollout_buffer.advantages))
+        logger.logkv("input_info/infoclip_range", clip_range)
+        if self.clip_range_vf is not None:
+            logger.logkv("input_info/clip_range_vf", clip_range_vf)
+        logger.logkv("input_info/old_log_probs", np.mean(old_log_probs))
+        logger.logkv("input_info/old_values", np.mean(old_values))
+
+        # Histograms
```

Comment:
please remove those, I don't think we need them, no?
(unless you are doing full debug)

Reply:
i don't see the real need for most users